documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -134,6 +144,14 @@ NOTE: Any user-specified OS file can bypass all prefixes by using an absolute pa
 
 See also <<overlay_prefix,`overlay_prefix`>> and <<upstream_kernel,`upstream_kernel`>>.
 
+==== `otg_mode` (Raspberry Pi 4 Only)
+
+USB On-The-Go (often abbreviated to OTG) is a feature that allows supporting USB devices with an appropriate OTG cable to configure themselves as USB hosts. On older Raspberry Pis, a single USB 2 controller was used in both USB host and device mode.
+
+Pi 4B and Pi 400 (not CM4 or CM4IO) add a high performance USB 3 controller, attached via PCIe, to drive the main USB ports. The legacy USB 2 controller is still available on the USB-C power connector for use as a device (`otg_mode=0`, the default).
+
+`otg_mode=1` requests that a more capable XHCI USB 2 controller is used as another host controller on that USB-C connector. Note that because CM4 and CM4IO don't include the external USB 3 controller, RPiOS images set `otg_mode=1` on CM4s for better performance.
```

Comment:
Another example where moving the side comment to a `[NOTE]` block would be helpful, as these get special handling by the renderer.